tests/functions/test_coalesce.py

```diff
@@ -7,6 +7,13 @@
 def test_coalesce():
     df = pd.DataFrame(
         {"a": [1, np.nan, 3], "b": [2, 3, 1], "c": [2, np.nan, 9]}
-    ).coalesce(["a", "b", "c"], "a")
+    ).coalesce(["a", "b", "c"], "d")
     assert df.shape == (3, 1)
     assert pd.isnull(df).sum().sum() == 0
+
+    df = pd.DataFrame(
```

Comment:
I think this should be a second test, rather than part of the first test? 

Perhaps distinguishing the two would be better too, such that it shows clearly that: 
- the first test gives everything a new column name, while 
- the second test takes the left-most column-name as the new column name.